Resources/doc/filter/multi_choice.rst

```diff
@@ -0,0 +1,124 @@
+======================
+Multiple choice filter
+======================
+
+This filter is very similar to choice filter, only you can select multiple options.
+It groups values of repository in a specified field, and returns available options.
```

Comment:
It groups values of a repository in a specified field and returns available options.